docs/pages/versions/unversioned/sdk/av.md

```diff
@@ -71,263 +66,6 @@ render() {
-
-      The `overrideFileExtensionAndroid` property may come in handy if the player receives an URL like `example.com/play` which redirects to `example.com/player.m3u8`. Setting this property to `m3u8` would allow the Android player to properly infer the content type of the media and use proper media file reader.
-
-    - `require('path/to/file')` for a media file asset in the source code directory.
-    - An [`Asset`](asset.md) object for a media file asset.
-
-    The [iOS developer documentation](https://developer.apple.com/library/ios/documentation/Miscellaneous/Conceptual/iPhoneOSTechOverview/MediaLayer/MediaLayer.html) lists the audio and video formats supported on iOS.
-
-    There are two sets of audio and video formats supported on Android: [formats supported by ExoPlayer](https://google.github.io/ExoPlayer/supported-formats.html) and [formats supported by Android's MediaPlayer](https://developer.android.com/guide/appendix/media-formats.html#formats-table). Expo uses ExoPlayer implementation by default; to use `MediaPlayer`, add `androidImplementation: 'MediaPlayer'` to the initial status of the AV object.
-
-  - **initialStatus (_AVPlaybackStatusToSet_)** -- The initial intended `AVPlaybackStatusToSet` of the `playbackObject`, whose values will override the default initial playback status. This value defaults to `{}` if no parameter is passed. See below for details on `AVPlaybackStatusToSet` and the default initial playback status.
-
-  - **downloadFirst (_boolean_)** -- If set to true, the system will attempt to download the resource to the device before loading. This value defaults to `true`. Note that at the moment, this will only work for `source`s of the form `require('path/to/file')` or `Asset` objects.
-
-  #### Returns
-
-  A `Promise` that is fulfilled with the `AVPlaybackStatus` of the `playbackObject` once it is loaded, or rejects if loading failed. The `Promise` will also reject if the `playbackObject` was already loaded. See below for details on `AVPlaybackStatus`.
-
-- `playbackObject.unloadAsync()`
-
-  Unloads the media from memory. `loadAsync()` must be called again in order to be able to play the media.
-
-  #### Returns
-
-  A `Promise` that is fulfilled with the `AVPlaybackStatus` of the `playbackObject` once it is unloaded, or rejects if unloading failed. See below for details on `AVPlaybackStatus`.
-
-- `playbackObject.getStatusAsync()`
-
-  Gets the `AVPlaybackStatus` of the `playbackObject`.
-
-  #### Returns
-
-  A `Promise` that is fulfilled with the `AVPlaybackStatus` of the `playbackObject`. See below for details on `AVPlaybackStatus`.
-
-- `playbackObject.setOnPlaybackStatusUpdate(onPlaybackStatusUpdate)`
```

Comment:
It might be an improvement, but currently we document this method in Audio and Video, so I think that we can leave current code as is, so the conversion do not include package code changes. 

This move can be made in a separate PR after the conversion lands, does it sounds good?